src/emulate.rs

```diff
@@ -746,6 +750,19 @@ fn exec_sra(state: &mut EmulatorState, rtype: RType) {
     state.pc_next();
 }
 
+// rd = 1                     ||| if (rs1 < rs2)
```

Comment:
nit-pick: Let's make the comment explicit about signedness by saying `||| if (rs1 <s rs2)` here.